src/main/java/com/google/devtools/build/lib/buildtool/ExecutionTool.java

```diff
@@ -993,11 +993,15 @@ public static void configureResourceManager(ResourceManager resourceMgr, BuildRe
     ImmutableMap<String, Double> cpuRam =
         ImmutableMap.of(
             ResourceSet.CPU,
+            // Replace with 1.0 * ResourceConverter.HOST_CPUS.get() after flag deprecation
             options.localCpuResources,
             ResourceSet.MEMORY,
+            // Replace with 0.67 * ResourceConverter.HOST_RAM.get() after flag deprecation
             options.localRamResources);
     ImmutableMap<String, Double> resources =
-        Stream.concat(options.localExtraResources.stream(), cpuRam.entrySet().stream())
+        Stream.concat(
+          Stream.concat(options.localExtraResources.stream(), cpuRam.entrySet().stream()),
+          options.localResources.stream())
```

Comment:
That would've been fine anyways since the deprecation would have happened in a major release where breaking changes are allowed. That being said, I'm fine with this for now.